spyder_kernels/customize/spyderpdb.py

```diff
@@ -322,8 +337,7 @@ def postcmd(self, stop, line):
 
         Is that good or too lazy? i.e. is more specific behaviour desired?
         """
-        if '!get_ipython().kernel' not in line:
```

Comment:
Why is this not necessary anymore?

Reply:
This was the old way of running cells and is not used anymore